alacritty_terminal/src/term/mod.rs

```diff
@@ -1694,6 +1703,7 @@ impl<T: EventListener> Handler for Term<T> {
         match mode {
             ansi::ClearMode::Above => {
                 let cursor = self.grid.cursor.point;
+                let prompt_mark = self.grid[cursor.line][Column(0)].flags & Flags::PROMPT_MARK;
```

Comment:
This seems really strange, there's gonna be nothing there anymore but the prompt line will still be set?